keras/utils/visualize_util.py

```diff
@@ -9,7 +9,15 @@
                        ' and graphviz for `pydotprint` to work.')
 
 
-def model_to_dot(model, show_shapes=False, show_layer_names=True):
+def model_to_dot(model, show_shapes=False, show_layer_names=True,
+                 verbose=False):
```

Comment:
`verbose` generally means something else (verbosity level in logs). Rename to `show_layer_attributes`.